gossipd/routing.c

```diff
@@ -948,6 +960,15 @@ bool routing_add_channel_update(struct routing_state *rstate,
 			      (flags & ROUTING_FLAGS_DISABLED) == 0, timestamp,
 			      htlc_minimum_msat);
 
+	/* For private channels, we get updates without an announce: don't
```

Comment:
A little undesirable for #1330: how will `invoice` query feerates for private channels?

Reply:
It could be an advantage though. We could have a `gossip_get_private_routes` message, which just iterates over the local channels and checks for `private_update` field.